tracer/src/Datadog.Trace/Ci/MethodSymbolResolver.cs

```diff
@@ -0,0 +1,288 @@
+// <copyright file="MethodSymbolResolver.cs" company="Datadog">
+// Unless explicitly stated otherwise all files in this repository are licensed under the Apache 2 License.
+// This product includes software developed at Datadog (https://www.datadoghq.com/). Copyright 2017 Datadog, Inc.
+// </copyright>
+
+using System;
+using System.Collections.Generic;
+using System.Reflection;
+using System.Runtime.CompilerServices;
+using Datadog.Trace.Logging;
+using Datadog.Trace.Util;
+using Datadog.Trace.Vendors.dnlib.DotNet;
+using Datadog.Trace.Vendors.dnlib.DotNet.Emit;
+using Datadog.Trace.Vendors.dnlib.DotNet.Pdb;
+
+namespace Datadog.Trace.Ci
+{
+    internal class MethodSymbolResolver
```

Comment:
NIT (as it's used here only for now anyway) but I wonder if this shouldn't be located in `Datadog.Trace.PDBs` that @OmerRaviv has introduced recently

Reply:
done in [d2d667c](https://github.com/DataDog/dd-trace-dotnet/pull/2596/commits/d2d667cde5da31665bb367fa7615532c10bdd839) :)